config/config-windows.yaml

```diff
@@ -26,7 +26,41 @@ receivers:
       swap:
       process:
 
-  # agent self-observability metrics
+  # scrape iis perf counters at 1m interval
+  windowsperfcounters/iis:
+    collection_interval: 1m
+    perfcounters:
+      - object: Web Service
+        instances: _Total
+        counters:
+          # TODO: instead of setting capture group values to lowercase, make this option explicit
+          - Current Connections
+          - Total Bytes received
+          - Total Bytes sent
+          - Total Connection Attempts (all instances)
+          - Total delete Requests
+          - Total get Requests
+          - Total head Requests
+          - Total options Requests
+          - Total post Requests
+          - Total put Requests
+          - Total trace Requests
+
+  # scrape mssql perf counters at 1m interval
```

Comment:
How does that work when MSSQL is not used / available in a particular use case? Will these counters be dropped?

Reply:
Yea very good question. Currently it would crash.

I was actually just writing a PR to change this so that warnings are logged on startup instead which is probably a better experience.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1538

Reply:
+1.